### PR TITLE
reorder carbon destroy to fix ghosting

### PR DESCRIPTION
**Pull request by optimumtact**
We cannot qdel these things until after our parent handling has
finished, so that we have things like dna available to the ghost
intialization from the body

### CODE REVIEW

**Comment by optimumtact** [comment]
I'd like to remove the ghostize from the Destroy, and make it such that is ghostize is not called before deletion of the mob then the client is kicked from the server (along with a runtime) (to make it really obvious where we have fucked up)

That way the path is fixed such that Death is called first, the mob is ghosted, then we can safely Destroy it.

Thoughts @tgstation/commit-access

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
+
 	QDEL_LIST(internal_organs)
 	QDEL_LIST(stomach_contents)
 	QDEL_LIST(bodyparts)
 	QDEL_LIST(implants)
 	remove_from_all_data_huds()
 	QDEL_NULL(dna)
-	return ..()
+
+	return .
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon.dm]: ???????????????????????????????????????????????????????

Comment (optimumtact) [reply code/modules/mob/living/carbon/carbon.dm]: explicit is better than implicit

Comment (duncathan) [reply code/modules/mob/living/carbon/carbon.dm]: better how? I think this is ugly af but maybe you mean something else

Comment (optimumtact) [reply code/modules/mob/living/carbon/carbon.dm]: it's better because it's obvious at first glance to everyone

Comment (lzimann) [reply code/modules/mob/living/carbon/carbon.dm]: if a person doesn't know what `. = ..()` is, then they really shouldn't be touching these things

**Comment by lzimann** [comment]
this will probably break something, did anyone test already?

**Comment by AnturK** [comment]
Did quick test, all seems fine.

**Comment by lzimann** [comment]
what could go wrong

**Pull request merged**